File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1192_ad_hoc.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>

int main() {
    int casos;
    scanf("%d", &casos);

    for (int i = 0; i < casos; i++) {
        int alg1, alg2;
        char letra;

        scanf("%d %c %d", &alg1, &letra, &alg2);

        int valor = 0;
        if (alg1 == alg2) {
            valor = alg1 * alg2;
        } else if (letra == toupper(letra)) {
            valor = alg2 - alg1;
        } else if (letra == tolower(letra)) {
            valor = alg1 + alg2;
        }

        printf("%d\n", valor);
    }
    return 0;
}

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1192_ad_hoc.js
import { question } from "readline-sync";

function main() {
    let quantidadeCasos = get_Number();
    for (let i = 0; i < quantidadeCasos; i++) {
        let [alg1, letra, alg2] = get_Text().split(``);
        let num1 = parseInt(alg1), num2 = parseInt(alg2);
        let valor;

        if (num1 === num2) {
            valor = num2 * num1;
        } else if (letra === letra.toLowerCase()) {
            valor = num1 + num2;
        } else if (letra === letra.toUpperCase()) {
            valor = num2 - num1;
        }
        
        print(valor);
    }
}

function get_Number() {
    return parseInt(question());
}

function get_Text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();